### ISSUE TITLE
the train of dcrnn is not working

### ISSUE THREAD

**Issue by Chrisable** [dmlc/dgl#4709]
## ❓ 100 epochs have not converged, but the setting is greater than 100 epochs. The initial valid mae and test mae loss directly run to 2.14 and 2.33. It is felt that these values are abnormal
![image](https://user-images.githubusercontent.com/49330390/195606567-9d59e6ba-dbbb-4ec8-a5b6-f11f0e7d5cee.png)

**Comment by czkkkkkk** [comment]
Hi, could you provide more details about what training script you are using and what are the training setting?

**Comment by czkkkkkk** [comment]
Hi Chrisable, I am closing this issue assuming you are happy about our response. Feel free to follow up and reopen the issue if you have more questions with regard to our response.
